### ISSUE TITLE
[Feature Request] Enable caching for pytest autocomplete

### ISSUE THREAD

**Issue by az-pz** [alfredodeza/zsh-plugins#3]
I have been using pytest plugin for pytest autocompletion. It takes few seconds to parse pytest help output and thus autocomplete is slow. Caching of pytest help output would make autocompletion much faster.

**Comment by alfredodeza** [comment]
I agree! I just don't know enough to try and make it work. At some point I had something but it was surely incorrect because I couldn't get any speed benefits :(

**Comment by az-pz** [comment]
I made it faster by storing the pytest help text in a location and reading from it instead of running `pytest --help` command. 
Much faster. Long live the hacky way.
